.ci/install_runtime.sh

```diff
@@ -162,3 +163,20 @@ if [ "$arch" == "aarch64" -a "${KATA_HYPERVISOR}" == "qemu" -a "${ENABLE_ARM64_U
 	#enable pflash
 	sudo sed -i 's|#pflashes|pflashes|' "${runtime_config_path}"
 fi
+
+if [ "$TEE_TYPE" == "tdx" ]; then
+        echo "Use tdx enabled guest config in ${runtime_config_path}"
+        sudo sed -i -e 's/vmlinux.container/vmlinuz-tdx.container/' "${runtime_config_path}"
+        sudo sed -i -e 's/^# confidential_guest/confidential_guest/' "${runtime_config_path}"
+        sudo sed -i -e 's/^kernel_params = "\(.*\)"/kernel_params = "\1 force_tdx_guest tdx_disable_filter "/g' "${runtime_config_path}"
+
+        case "${KATA_HYPERVISOR}" in
+		"cloud-hypervisor")
+			sudo sed -i -e 's/^firmware = ".*"/firmware = "\/usr\/share\/td-shim\/final-pe.bin"/' "${runtime_config_path}"
+			;;
+
+		"qemu")
+			sudo sed -i -e 's/^firmware = ".*"/firmware = "\/usr\/share\/qemu\/OVMF.fd"/' "${runtime_config_path}"
```

Comment:
I think `firmware_volume` is also needed, isn't it?

Reply:
Yes, it is optional, but we should also add it.